tests/test_notebooks.py

```diff
@@ -62,6 +62,11 @@ def test_impactplot(common_kwargs):
     pm.execute_notebook('docs/examples/notebooks/ImpactPlot.ipynb', **common_kwargs)
 
 
+@pytest.mark.slow
```

Comment:
Do we still need this anymore? Perhaps so if we want to be able to run all the notebooks locally? Not sure.